packages/core/src/awsService/ec2/explorer/ec2InstanceNode.ts

```diff
@@ -100,3 +101,14 @@ export class Ec2InstanceNode extends AWSTreeNodeBase implements AWSResourceNode
         await vscode.commands.executeCommand('aws.refreshAwsExplorerNode', this)
     }
 }
+
+export async function refreshExplorerNode(node?: Ec2Node) {
```

Comment:
I think we have a generalized function somewhere like this already.

Reply:
Yea, this is just a wrapper around that function that makes sure the node exists. Avoids having the same logic in each command where node is optional.